webserver/pkgpkr/static/styles.css

```diff
@@ -186,6 +193,41 @@ body {
   border-color: transparent transparent #555 transparent;
 }
 
+.tooltip-recommendtion {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltip-recommendtion .tooltiptext-recommendtion {
+  visibility: hidden;
+  min-width: 120px;
+  max-width: 100%;
+  background-color: #555;
+  color: #fff;
+  border-radius: 6px;
+  padding: 5px 10px 10px 10px;
+  position: absolute;
+  z-index: 1;
+  top: 125%;
+  left: 50%;
+  margin-left: -60px;
```

Comment:
Negative margins are generally to be avoided if possible.

Reply:
I will try to figure out how to replace it in the next branch